tests/fixtures/simulation.yml

```diff
@@ -128,6 +128,22 @@
   contains:
     - "[Simulator] Running ./vendor/bin/drush -y --root=$(pwd)/build --site-name='Site name' --site-mail=info@example.org --locale=en --account-mail=admin@example.org --account-name=admin --account-pass=admin --sites-subdir=default --db-url='mysql://root:root@127.0.0.1:3306/drupal' site-install standard"
 
+- command: 'drupal:site-install --database-type=sqlite3'
```

Comment:
`sqlite3` is not a valid database type, the version is inconsequential. We also don't pass `mysql-8.0.11`.